docs/concepts/storage/persistent-volumes.md

```diff
@@ -512,6 +539,17 @@ parameters:
 
   When persistent volumes are dynamically provisioned, the Gluster plugin automatically creates an endpoint and a headless service in the name `gluster-dynamic-<claimname>`. The dynamic endpoint and service are automatically deleted when the persistent volume claim is deleted.
 
+#### HostPath
+
+```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: standard
+provisioner: kubernetes.io/host-path
+```
+__Important!__ HostPath provisioner can be used for single node testing only. DO NOT USE it in your production environment.
```

Comment:
IMO we should remove HostPath from here. Even though this warning is present, people will still try it on multi-node clusters and see that it deceptively appears to "work." When in reality it will break down under any situation involving scheduling complications, data needs to be moved, etc.